src/App.tsx

```diff
@@ -168,6 +173,23 @@ export default function App() {
       addLog("[error] signMessage: " + JSON.stringify(err));
     }
   };
+  const signAndSendTransaction = async () => {
+    try {
+      const transaction = await createTransferTransaction();
+      if (!transaction) return;
+      const { signature } = await provider.signAndSendTransaction(transaction);
```

Comment:
Just an FYI, all our sample code should probably be using `signAndSendTransaction` unless it is specifically demonstrating those other methods. That was the case at some point but I think it got removed.

Reply:
This sandbox code is mostly demonstrating the other methods, should I remove the other sign methods other than `signAndSendTransaction`?

Reply:
Hi i am developing a mint site with phantom and sol. On devnet when I send sol to another wallet the transaction get confirmed but on the reciever end i don't get the sols. So what will be the reason? I am following this

https://codesandbox.io/s/nifty-lewin-4bvd8?file=/src/App.tsx